README.md

```diff
@@ -1,5 +1,30 @@
 # Onigumo #
 
+## About
```

Comment:
```suggestion
## About ##
```

Pouze pro konzistenci.

Reply:
Yes, but I dont know why do you use two hastags twice, at the beginning and also at the end. I supposed the position at the beginning is enough. There is more work in you approach and it is unnecessary to make heading

Reply:
I know. But… can we keep it consistent for now and retract to the more space savvy version later? 🥹